payment-api/PaymentApi/Features/CreatePaymentSetupIntent.cs

```diff
@@ -23,20 +23,20 @@ public class CommandValidator : AbstractValidator<Command>
         {
             public CommandValidator()
             {
-                RuleFor(x => x.UserId)
-                    .NotEmpty();
+                //RuleFor(x => x.UserId)
```

Comment:
Cann we pass them as parameters in request instead?

Reply:
I do not get how these are used atm, for example the /payment-intent post does not have this body, so I am not sure where to set set them

Reply:
Ok. I will update the request method parameters later.